From b830f6b2bfb3ecd407947de060d59def641997d1 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Mon, 9 Apr 2018 11:35:20 -0700
Subject: [PATCH 6/6] Don't print out pointers for no reason.

---
 plugins/duktape.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/plugins/duktape.c b/plugins/duktape.c
index 0d54e72..95caef8 100644
--- a/plugins/duktape.c
+++ b/plugins/duktape.c
@@ -135,7 +135,6 @@ static int create_object(struct pacrunner_proxy *proxy)
 	}
 
 	__pacrunner_proxy_set_jsctx(proxy, ctx);
-	pacrunner_error("done %p\n", ctx);
 	return 0;
 }
 
-- 
2.17.0

